docs/utils/tap.rst

```diff
@@ -445,6 +445,15 @@ It is possible to use a file where the credentials are stored:
   >>> gaia.login(credentials_file='my_credentials_file')
 
 
+If you do not provide any parameters at all, a prompt will ask for user name and password.
```

Comment:
Is this whole document a duplicate of the above document?

Reply:
Yes, I think that's better when the code is _nearly_ the same.  But I don't feel strongly about this.